### PR TITLE
mobile menu

### PR DESCRIPTION
**Pull request by FevenSeyfu**
Added mobile menu for smaller screens, milestones

- open the mobile menu when the hamburger icon clicked

- close the mobile menu when the close icon clicked

- go to a particular section based on the selected navbar item

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,115 +22,127 @@
...
+				<img src="images/icons/Cancel-Icon.svg" alt="cancel button icon" class="cancel-icon">
+			</button>
+			<div class="main-navbar">
+				<ul class="nav-items mobile">
+					<li><a href="#portolio" class="nav-item mobile portfolio" >Portfolio</a></li>
+					<li><a href="#about" class="nav-item mobile about">About</a></li>
+					<li><a href="#contact" class="nav-item mobile contact">Contact</a></li>
+				</ul>
+			</div>
+		</div>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Your Top elements(Nav elements) should be inside the `<header />` tag and your `Navigation` menu items should be inside the `<nav />` tag. Please implement my suggestions 🙏🏼 

```HTML
<header>
    <h1>Website Title</h1>
    <nav>
        <!-- Main Navigation inside <header> -->
        <ul>
            <li></li>
        </ul>
    </nav>
</header>
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @FevenSeyfu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [ ] [OPTIONAL] It would be nice if you placed your codebase prettier 👍🏼 

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (FevenSeyfu) [reply index.html]: Thank you for your feedback, I have amended the code and added the top section inside the header tag and I have also used the nav tag for the mobile menu.

**Review by jerryowusu — APPROVED** [review]
Hi @FevenSeyfu  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

good job


## Highlights
All linters are passing :heavy_check_mark: 
Made the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @FevenSeyfu, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Well-implemented suggestions by the previous reviewers.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**